index.html

```diff
@@ -0,0 +1,282 @@
+      <div id="about-myself">
+        <h3>About<br> mySelf</h3>
+        <p>
+          I'm a software developer! I can help you build a product , feature
+          or website Look through some of my work and experience! If you like
+          what you see and have a project you need coded, don't hestiate to
+          contact me.
+        </p>
+
+        <a class="pointer btn-action">LET'S CONNECT</a>
+
+        <ul class="socials">
+          <li class="pointer">
+            <img src="./images/twitter.svg" alt="Twitter" />
+          </li>
+          <li class="pointer">
+            <img src="./images/Linkedin.svg" alt="Linkedin" />
+          </li>
+          <li class="pointer">
+            <img src="./images/medium.svg" alt="Medium" />
+          </li>
+          <li class="pointer">
+            <img src="./images/github.svg" alt="Github" />
+          </li>
+          <li class="pointer">
+            <img src="./images/angellist.svg" alt="Angellist" />
+          </li>
+        </ul>
+        <a class="btn btn-action">Get my resume</a>
+      </div>
+
+      <div id="about-myself-2">
+        <ul id="lfs-section">
+          <li id="languages">
+            <h4>
+              Languages
+              <img src="./images/down.svg" alt="Arrow" />
+            </h4>
+            <ul>
+              <li>
+                <img src="./images/js.png" alt="js" />
+                <h6>JavaScript</h6>
+              </li>
+              <li>
+                <img src="./images/html5.png" alt="html" />
+                <h6>HTML</h6>
+              </li>
+              <li>
+                <img src="./images/css3.png" alt="css" />
+                <h6>CSS</h6>
+              </li>
+            </ul>
+          </li>
+          <li id="frameworks">
+            <h4>
+              Frameworks
+              <img src="./images/Icon-Chevron-Right-Larger.png" alt="Arrow" />
+            </h4>
+            <ul></ul>
+          </li>
+          <li id="skills">
+            <h4>
+              Skills
+              <img src="./images/Icon-Chevron-Right-Larger.png" alt="Arrow" />
+            </h4>
+            <ul></ul>
+          </li>
+        </ul>
+      </div>
+    </section>
+    <section id="contact">
+      <div>
+        <h2>Contact me</h2>
+        <p>
+          If you have an application you are interested in developing, a
+          feature that you need built or a project that needs coding. I'd love
+          to help with it
+        </p>
+      </div>
+
+      <form action="https://formspree.io/f/mlezyjdr" method="post">
+        <input name="Name" type="text" placeholder="Input your name" maxlength="30" required />
+        <input name="Email" type="email" placeholder="Input your email" required />
+        <textarea name="message" id="message" placeholder="Write your massege here" maxlength="500" required></textarea>
+        <button type="submit">Get in touch</button>
+      </form>
+    </section>
+  </body>
+
+</html>
```

Comment:
- You did a great job by adding the contact section but Kindly remove it as it has not yet the part of the requirement. you will add it to the next project with a better understanding of the lesson.  :+1: :+1: